demo/app/lib/wallet_sdk/wallet_sdk_mobile.dart

```diff
@@ -211,6 +211,12 @@ class WalletSDK extends WalletPlatform {
     return null;
   }
 
+  Future<List<IssuerMetaData>> getIssuerMetaData() async {
```

Comment:
Please add getIssuerMetaData() stub to wallet_sdk_js.dart in otherwise it will break the web version build.

Reply:
done